### ISSUE TITLE
Cannot create new plot areas due to overlapping with an existing area.

### ISSUE THREAD

**Issue by godnroc** [IntellectualSites/PlotSquared#2511]
__*NOTICE: Bukkit/Spigot versions 1.7.10 to 1.12.2 are considered legacy and will receive limited support. Please consider upgrading to 1.13 for future support. Plugins exist for 1.13+ that bring back old behaviors found in 1.8*__
# Bug report template 

**Debug paste link:** 

**[REQUIRED] Spigot/Paper Version Number:** git-Spigot-065a373-03145b

**[REQUIRED] Minecraft Version Number:** 1.14.4

**[REQUIRED] Description of the problem:** When creating a new plot area in an existing world, an error stating that the area could not be created due to an overlap occurs. 

**Any relevant console output or screenshots:**
![Plots](https://user-images.githubusercontent.com/2201722/65380646-3c3f3280-dca5-11e9-8605-fb879d6d1151.png)
![Details](https://user-images.githubusercontent.com/2201722/65380647-46f9c780-dca5-11e9-9f29-ac4d0c51ec62.png)

**Plugins being used on the server:** LuckPerms, WorldEdit, TreeCapitor, Vault, WorldEditSelectionVisualizer, EssentialsX, Plot Squared, Multiverse-Core, Multiverse-Portals, Multiverse-NetherPortals

**How to replicate:**
- In the above screenshot, the southeast corner was create first.
- Following that, the southern strip was created without issue.
- The southwestern strip was then created.
- After this, the western and eastern strips could not be created due to overlaps with the southwestern and southeastern corners respectively.
- The northwestern and northeastern corners were created without issue.
- The northern strip could no be created due to an overlap with the southern strip.

- Each area was created with the following command, altered for the proper name:
> /plot area create city:north type=2 size=64 gap=32 main=dirt floor=grass_block border=smooth_stone_slab
-The areas were created to be aligned to the games chunks by selecting the outermost corner of the chunks which defined the corners of the area.
- Each of the gaps between areas were worldedited'd to match the plot area paths.

**Checklist**:

- [X] I included all information required in the sections above
- [X] I made sure there are no duplicates of this report [(Use Search)](https://github.com/IntellectualSites/PlotSquared/issues?utf8=%E2%9C%93&q=is%3Aissue)
- [X] I made sure I am using an up-to-date version of PlotSquared
- [X] I made sure the bug/error is not caused by any other plugin

**Comment by AeSix** [comment]
Are you trying to create a plot area within a plot world?  I'm not sure this is possible.  You would need to create only plot areas within a flat world to achieve this.   If you have documentation stating otherwise, please join the discord server to discuss it.

**Comment by godnroc** [comment]
I originally posted in the discord, was told to post here instead. In response to AeSix, I posted again in the discord and received no response.

![Discord to Github](https://user-images.githubusercontent.com/2201722/66361793-cd6d0500-e945-11e9-9e79-24779c39ad6c.PNG)

To answer your question: no, the world in question is not a plot world, but a flat world that I began manually creating plots with WorldEdit before finding this plugin. I've been away from plugins for a long time, and am still in the mindset of manually defining every plot. The areas in blue in my screenshot are vestigial from my original, manual attempts.

This plugin seems to be FAR more efficient that what I am use to, but not being able to create plots in the three areas in the original post does quite ruin it for me.

Should I continue here or on the Discord server?

**Comment by AeSix** [comment]
Much more information now, important information that you did not originally provide.  Had you provided that information originally, my response would had been the same as what I'm about to tell you.

In this case, it would be a lot easier on you, now and in the long run, to create a new world for PlotSquared's plots.  Yes, you will have to manually copy/paste your players plots from the old world to the new world.  But honestly, many players may choose not to copy them.  Simply put an End of Life announcement out (which ever means is best to reach your players) regarding the old plot world.  You'll find many players won't care to copy the things over - especially if you give them a month or so before EoL'ing the old world.  

The advantage to this is that PlotSquared will be able to create the virgin world without any legacy issues being inherited.  You won't have the above difficulties, and your plots with be much more sane. 

PlotSquared can't create plots in already protected areas. 

The only way I see to migrate the entire world to PlotSquared, and that's only if your manual grid lines up exactly from edge to edge (which it appears not to?)  That's to remove your current plot system (worldguard regions, I'm guessing?), configure P2 to be exactly the same as the layout, and then manually assign plots to the player who owned the space previously.  If you're a developer, you might be able to write something to convert your system to P2, and would need to be done before you remove the current protection system, but also after the P2 world and database has been created.  i.e. a lot of work. 

As for creating three separate zones of plots in a single world, you can do this.  See https://github.com/IntellectualSites/PlotSquared/wiki/Augmented-Plot-Worlds
I do not know what the exact steps are for 1.13.2+ are, and apparently that page has been deprecated.  You *may* be able to create the areas in 1.12.2, and then upgrade PlotSquared and Spigot to 1.13.2.   Otherwise, it may be better to create 3 different worlds.  You can do that with PlotSquared, or with Multiverse.  I prefer using PlotSquared for that, but Multiverse works perfectly well for many others, from 1.12 through 1.14.4 at least.  Alternatively, you could have a single world, and then claim every plot from center out to the north, west and east, merge them, set that plot to be a server plot.  This would give the allusion of three sectors, but the world would still be a single plot area.  I don't know if that would work for your needs, as I can imagine scenarios where those three zones have different functions that would be lost in a flat plot area. 

Unfortunately, your setup is far more niche than is something that can be handled by the developers, or to be expected to be converted to PlotSquared automatically.  Again, I suggest EoL'ing the current world and move players to a new world (or three)
